vitest.setup.ts

```diff
@@ -11,6 +11,25 @@ const fetchMock = createFetchMock(vi)
 // sets globalThis.fetch and globalThis.fetchMock to our mocked version
 fetchMock.enableMocks()
 
+const cookies = reactive<{ [key: string]: any }>({})
+interface CookieOptions {
+  default?: () => any
+}
+
+export const useCookieMock = vi.fn((key: string, opts?: CookieOptions) => {
+  const cookie = toRef(cookies, key)
+  if (cookie.value === undefined && opts?.default) {
```

Comment:
What happens if both `cookie` and `opts.default` don't have values? Won't we be returning an undefined ref?

Reply:
I think that's actually just how regular cookies work anyway.